static/src/js/components/GranuleResults/GranuleResultsDownloadNotebookButton.jsx

```diff
@@ -101,8 +103,15 @@ export const GranuleResultsDownloadNotebookButton = ({
           <Dropdown.Menu
             ref={dropdownMenuRef}
             className="granule-results-download-notebook-button__menu"
+            data-testid="dropdown-menu"
```

Comment:
We should avoid using test ids. In this example I don't think it is necessary

```
        const dropdownMenu = screen.getByTestId('dropdown-menu')
        const downloadButton = within(dropdownMenu).getByRole('button', { name: 'Download Notebook' })
```

Does `screen.getByRole('button', { name: 'Download Notebook' })` work without needing the test id?